setup.py

```diff
@@ -46,7 +46,10 @@
         "mbed-devices~=1.0",
         "mbed-project~=2.0",
         "pdoc3",
```

Comment:
"merge mbed_tools_lib package" commit title should start with a capital letter.

Reply:
Which commit were these copied from? Note the source repo and commit in the commit message.

Reply:
these were pre-existing here, this commit removes `mbed_tools_lib` (below) as this is the commit that merges it.